python_modules/dagster/dagster/core/storage/sqlite.py

```diff
@@ -1,37 +1,8 @@
 import os
 import sqlite3
-from functools import update_wrapper
 
 from dagster import check
 
-from .sql import run_migrations_offline as run_migrations_offline_
-from .sql import run_migrations_online as run_migrations_online_
-
-
-def run_migrations_offline(*args, **kwargs):
```

Comment:
we kind of copy/pasted this code for each of our alembic directories, this is just a consolidation removal.